product_docs/docs/pgd/5/terminology.mdx

```diff
@@ -2,83 +2,116 @@
 
-Conflict-free replication with all cluster members. Technically, this is synchronous logical replication using two-phase commit (2PC).
+
+#### Eager
+
+Eager is a synchronous commit mode which avoids conflicts by detecting incoming potentially conflicting transactions and “eagerly” one of aborting them to maintain consistency. 
 
 #### Eventual consistency
 
-A distributed computing consistency model stating changes to the same item in different cluster members will converge to the same value. With PGD, this is achieved through asynchronous logical replication with conflict resolution and conflict-free replicated data types.
+A distributed computing consistency model stating changes to the same item in different cluster members will eventually converge to the same value. Asynchronous logical replication with conflict resolution and conflict-free replicated data types exhibit eventual consistency in PGD.
 
 #### Failover
 
-The automated process that recognizes a failure in a highly available database cluster and takes action to connect the application to another active database. The goal is to minimize downtime and data loss.
+The automated process that recognizes a failure in a highly available database cluster and takes action to maintain consistency and availability. The goal is to minimize downtime and data loss.
 
-#### Horizontal scaling or scale out
+#### Group commit
 
-A modern distributed computing approach that manages workloads across multiple nodes, such as scaling out a web server to handle increased traffic.
+A synchronous commit mode which requires more than one PGD node to successfully receive and confirm a transaction at commit time.
 
-#### Logical replication
+#### Immediate consistency
 
-Provides more flexibility than physical replication in terms of selecting the data replicated between databases in a cluster. Also important is that cluster members can be on different versions of the database software.
+A distributed computing model where all replicas are updated synchronously and simultaneously. This ensures that all reads after a write has completed will see the same value on all nodes. The downside of this approach is its negative impact on performance.
 
-#### Nines
+#### Logical replication
 
-A measure of availability expressed as a percentage of uptime in a given year. Three nines (99.9%) allows for 43.83 minutes of downtime per month. Four nines (99.99%) allows for 4.38 minutes of downtime per month. Five nines (99.999%) allows for 26.3 seconds of downtime per month.
+A more efficient method of replicating changes in the database. While physical streaming replication duplicate the originating database’s disk blocks, logical replication instead sees the DML commands - inserts, deletes and updates,- published to all systems that have subscribed to see the changes. Each subscriber then applies the changes locally. Logical replication is not able to support most DDL
```

Comment:
"sees the DML commands" may lead to the misinterpretation of using SQL- and trigger based replication, which have their own set of gotchas. And for which other OSS solutions for Postgres exist and are known problematic.

I'd like to ensure we avoid this confusion. How about: "logical replication instead transmits the changes independent of the underlying physical storage format"?

Reply:
```suggestion
A more efficient method of replicating changes in the database. While physical streaming replication duplicate the originating database’s disk blocks, logical replication instead takes the changes made, independent of the underlying physical storage format, and publishes them to all systems that have subscribed to see the changes. Each subscriber then applies the changes locally. Logical replication is not able to support most DDL
```